bindings_ffi/src/mls.rs

```diff
@@ -116,9 +116,16 @@ pub async fn connect_to_backend(
     Ok(Arc::new(XmtpApiClient(api_client)))
 }
 
+#[uniffi::export]
+pub fn is_connected(api: Arc<XmtpApiClient>) -> bool {
```

Comment:
is it possible this returns false negatives, for instance if the channel has just used up all its permits b/c its sending a bunch of requests at once? in that case, we'd have lots of requests pending but this might return not connected as a result

Reply:
Okay I think I resolved that in https://github.com/xmtp/libxmtp/pull/2172/commits/de416eaee74838c8719a1580155bf39a4cbf1066 by making it async.